spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/features/TaskConfiguration.java

```diff
@@ -211,6 +214,19 @@ public TaskAppDeploymentRequestCreator taskAppDeploymentRequestCreator(
 				metadataResolver, dataflowServerUri);
 	}
 
+	@Bean
+	public TaskRepository taskRepository(DataSource dataSource) {
+		MultiSchemaTaskExecutionDaoFactoryBean taskExecutionDaoFactoryBean = new MultiSchemaTaskExecutionDaoFactoryBean(dataSource, "TASK_");
```

Comment:
So will the MultiSchema* components be removed when the aggregate is removed? Those were invented for the aggregate multi version support.

Reply:
Correct.   Working toward their retirement.